atlas/AtlasMl/atlasml/ml/embeddings.py

```
@@ -0,0 +1,141 @@
+import os
+from typing import List, Tuple, Optional, Union
```

Ruff (0.12.2)

`typing.List` is deprecated, use `list` instead

(UP035)